compiler/types.nim

```diff
@@ -1471,6 +1471,41 @@ proc skipHiddenSubConv*(n: PNode; g: ModuleGraph; idgen: IdGenerator): PNode =
   else:
     result = n
 
+proc callConvMismatch*(formal, actual: PType, indented = false): string =
```

Comment:
why not always indent instead? even from looking at your tests, it'd look better